com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -10,11 +10,15 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 ### Added
 
+
 ### Fixed
-- Fixed a failing UTP test that was failing when you install Unity Transport package 2.0.0 or newer.
+
+- Fixed issue where `NetworkObject.SpawnWithObservers` was not being honored for late joining clients. (#2623)
+- Fixed a failing UTP test that was failing when you install Unity Transport package 2.0.0 or newer. (#2616)
```

Comment:
"Not your change", heh... but...

I asked Frank to remove this changelog in the release PR, can we remove it here too since you're touching it anyway? I don't generally think we need to broadcast test changes.